src/test/resources/syntheticsupervariables/input/com/example/Dog.java

```diff
@@ -0,0 +1,14 @@
+package com.example;
+import org.wild.Mammal;
+
+public class Dog extends Mammal {
+    String habitat;
+    boolean canBreathUnderWater;
+    public void Dog() {
```

Comment:
Is this supposed to be a constructor? If so, it's not correct: constructors should not have a return type, but this one has `void`.